### PR TITLE
Add support for WS2812 NeoPixel strips/arrays

### PR DESCRIPTION
**Pull request by rm-hull**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,15 +14,15 @@
...
     long_description="\n\n".join([README, CONTRIB, CHANGES]),
     license="MIT",
-    keywords=["raspberry pi", "rpi", "led", "max7219", "matrix", "seven segment", "7 segment"],
+    keywords=["raspberry pi", "rpi", "led", "max7219", "matrix", "seven segment", "7 segment", "neopixel", "ws2812", "ws281x"],
     url="https://github.com/rm-hull/luma.led_matrix",
     download_url="https://github.com/rm-hull/luma.led_matrix/tarball/" + version,
     namespace_packages=["luma"],
     packages=["luma.led_matrix"],
-    install_requires=["luma.core>=0.1.11"],
+    install_requires=["luma.core>=0.1.11", "ws2812"],
```
Comment (thijstriemstra) [inline setup.py]: @rm-hull this should probably be optional?

Comment (rm-hull) [inline setup.py]: In the general case: what are the trade-offs/benefits of making it optional, as opposed to auto-installing it as a dependency?

Comment (thijstriemstra) [inline setup.py]: well, if 5% of the people want to use ws2812 than 95% is presented with an extra dependency they don't need.

Comment (thijstriemstra) [inline setup.py]: The coverage also decreased with almost 9%, merging it with as the current or more coverage would be nice.

Comment (thijstriemstra) [inline setup.py]: I looked into the extra_requires and it seems unnecessary complex, so +1 for this.

Comment (rm-hull) [inline setup.py]: > The coverage also decreased with almost 9%

Yeah, I need to add tests (but also need to take a break 8-S)

Comment (thijstriemstra) [inline setup.py]: I saw the separate ticket for the tests, don't worry about it :)

**Pull request merged**